src/sidebar/components/Annotation/test/AnnotationHeader-test.js

```diff
@@ -443,6 +452,52 @@ describe('AnnotationHeader', () => {
     });
   });
 
+  context('when page_numbers feature is enabled', () => {
+    beforeEach(() => {
+      activeFeatures.page_numbers = true;
+
+      // Un-mock the `pageLabel` function.
+      $imports.$restore({
+        '../../helpers/annotation-metadata': true,
+      });
```

Comment:
We could have mocked `pageLabel`, but I think it is more straightforward just to pass an annotation with a real `PageSelector`.